awx/ui_next/src/screens/Job/constants.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable-next-line import/prefer-default-export */
+export const JOB_TYPE_URLS = {
```

Comment:
I learned the other day that there's actually a specific name for these - url path segment (or just url segment). 

How do you feel about `JOB_TYPE_URL_SEGMENTS` instead?